client/app/components/Paginator/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+// Styles
+import bulma from 'styles/bulma.scss';
+import styles from './styles.scss';
+
+// Components
+import RangeButtons from './RangeButtons';
+
+const Paginator = ({ currPage, numPages, getPosts }) => {
+  // Wrapper function to prevent out of range page requests
+  const getPostsSafe = (target) => {
+    if (target > 0 && target <= numPages) {
+      getPosts(target);
+    }
```

Comment:
Yea, doesn't have to be a hard break but, some kind of "oops"... good practice to have.